internal/testsdetection/testsdetection.go

```diff
@@ -7,6 +7,11 @@ import (
 
 var integrationtests = false
 
+// IsIntegrationTest returns true if we are running under integration tests.
+func IsIntegrationTest() bool {
+	return integrationtests
+}
```

Comment:
Where is this used in this commit?

Reply:
It was used in a previous version but not anymore.

Reply:
Removed